main.go

```diff
@@ -321,12 +317,34 @@ func main() {
 	}
 
 	setupLog.Info("starting manager", "version", version.Get().String())
-	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
+	if err := mgr.Start(ctx); err != nil {
```

Comment:
We had just called `ctrl.SetupSignalHandler()` a few lines above, so I assume this code should have been using that `ctx` rather than calling the func again. That's how CAPI does it.